### ISSUE TITLE
Vector subscript out of range in debug mode

### ISSUE THREAD

**Issue by bratao** [hiroshi-manabe/CRFSegmenter#4]
Hello,

Using the latest version(converter-crf-integration branch), in debug mode, I get a Vector subscript out of range at Line 134 at generatePatternSetSequence::InternalDataSequence.cpp

The line is 
` if (!possibleLabelSetList[pos - i].empty() && possibleLabelSetList[pos - i].find(seq.getLabelAt(i)) == possibleLabelSetList[pos - i].end()) {
                        labelsOK = false;
                        break;
                    }`

pos-1 was equal to 0 , but possibleLabelSetList was empty.

In release mode, the std don't check this

Any file would trigger this bug when --tag. But an example is:
```

0		B-tipo	1:*	1:token/0
1		I-tipo	1:*	1:token/1
2		I-tipo	1:*	1:token/2
3		I-tipo	1:*	1:token/3
```

**Comment by hiroshi-manabe** [comment]
Thank you for trying the latest version!
I think I fixed the problem in 16981fc.

**Comment by bratao** [comment]
@hiroshi-manabe , thank you. That was it !
